pydrive2/test/test_file.py

```diff
@@ -842,6 +842,39 @@ def test_Parallel_Insert_File_Passed_HTTP(self):
         files = self._parallel_uploader(self.FILE_UPLOAD_COUNT, 10)
         self._parallel_downloader(files, 10)
 
+    # Tests for Copy file.
+    # ====================
+
+    def test_CopyFile(self):
+        drive = GoogleDrive(self.ga)
+        file1 = drive.CreateFile()
+        filename = self.getTempFile("copytestfile")
```

Comment:
`content="hello world"` should be passed here instead of the next line and the `SetContentString`

Reply:
@simone-viozzi please check if we can do this ^^

Reply:
with 
```python
filename = self.getTempFile("copytestfile", content)
```

the test fails because `file1` has no content

Reply:
it should be `filename = self.getTempFile("copytestfile", content=content)` I think

if it doesn't work - it should be fixed I think

Reply:
I copied the usage on other tests:

https://github.com/iterative/PyDrive2/blob/2e43e4561d965ce78dc158a02fbdb75ba6c38105/pydrive2/test/test_file.py#L84-L89

but now i searched more and found a usage with content:

https://github.com/iterative/PyDrive2/blob/2e43e4561d965ce78dc158a02fbdb75ba6c38105/pydrive2/test/test_file.py#L222-L230

so to use content on getTempFile i need to:

```python
content = "hello world!"

# create a temp file and set it's content to a known string
file1 = drive.CreateFile()
filename = self.getTempFile("copytestfile", content=content)
file1["title"] = filename
file1.SetContentFile(filename)
pydrive_retry(file1.Upload)
```

And with `SetContentFile` the test runs.

i will update the code than.